packages/hawtio/src/plugins/jmx/index.ts

```diff
@@ -8,7 +8,7 @@ export const jmx = () => {
   hawtio.addPlugin({
     id: 'jmx',
     title: 'JMX',
-    path: '/jmx',
+    path: 'jmx/*',
```

Comment:
if react router v6 requires `/*` it shouldn't be configured here in each plugin. it should be a ui page concern so should be implemented in `HawtioPage.tsx`. A plugin shouldn't be aware of such impl details.